src/v/kafka/client/fetch_session.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+#include "kafka/client/fetch_session.h"
+
+#include "kafka/protocol/fetch.h"
+
+#include <fmt/ostream.h>
+
+#include <iostream>
+
+namespace kafka::client {
+
+model::offset fetch_session::offset(model::topic_partition_view tpv) const {
```

Comment:
topic_partition_view is ~100 bytes. we might want to provide a const-ref of `topic-partition` overload as well.

Reply:
Are you sure? I thought it was a `byte*`, `size_t`, and `int32_t`

Reply:
ah you are right! sorry, must have changed